core/connection.js

```diff
@@ -47,79 +47,79 @@ Blockly.Connection = function(source, type) {
 /**
  * Constants for checking whether two connections are compatible.
  */
-Blockly.Connection.CAN_CONNECT = 0;
-Blockly.Connection.REASON_SELF_CONNECTION = 1;
-Blockly.Connection.REASON_WRONG_TYPE = 2;
-Blockly.Connection.REASON_TARGET_NULL = 3;
-Blockly.Connection.REASON_CHECKS_FAILED = 4;
-Blockly.Connection.REASON_DIFFERENT_WORKSPACES = 5;
-Blockly.Connection.REASON_SHADOW_PARENT = 6;
-Blockly.Connection.REASON_DRAG_CHECKS_FAILED = 7;
+Connection.CAN_CONNECT = 0;
```

Comment:
This looks like an instance of "Exports a class constructor with unrelated static properties", which is one of the triage categories in https://github.com/google/blockly/issues/5073

Intuitively, I expected that these would be exported directly.

Reply:
Yes: not for this PR, but this looks like it should be an enum.  In that form I think it is probably closely-related enough that it could be exported either as a static property of `Connection` (e.g. `Connection.Reason`) or beside it (e.g., `exports = {Connection, ConnectionReason}`).  Added to list in #5073.

Reply:
Is there any action needed here beyond adding to the triage list?

Reply:
Nope.